File: JS/leetcode/跳跃游戏/1.js
/**
 * @跳跃游戏1
 */
let nums = [2,3,1,1,4]
//dp
let n=nums.length
let dp=new Array(n).fill(false)
dp[0]=true
for(let i=1;i<n;i++){
    for(let j=0;j<i;j++){
        if(dp[j]===true && nums[j]>=(i-j)){
            dp[i]=true
            break
        }
    }
}
console.log(dp)

File: python/leetcode/hot100/1.py
"""
两数之和
lc 1
"""
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums_map={}
        for index,num in enumerate(nums):
            deleteNum=target-num
            if deleteNum in nums_map:
                return [index,nums.index(deleteNum)]
            else:
                nums_map[num]=index